core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -1405,8 +1390,6 @@ class SparkContext(config: SparkConf) extends Logging with ExecutorAllocationCli
    */
   @DeveloperApi
   override def requestExecutors(numAdditionalExecutors: Int): Boolean = {
```

Comment:
outdated comments for this function? https://github.com/apache/spark/pull/7532/files#diff-364713d7776956cb8b0a771e9b62f82dR1389